composables/track.ts

```diff
@@ -1,8 +1,22 @@
 import { TrackApi } from "@bcc-code/bmm-sdk-fetch";
-import type { TrackGetRequest } from "@bcc-code/bmm-sdk-fetch";
+import type {
+  TrackGetRequest,
+  TrackIdGetRequest,
+} from "@bcc-code/bmm-sdk-fetch";
 
 export function useTracks(options: TrackGetRequest = {}) {
   return reactiveApi(
     useLazyAsyncData("tracks", () => new TrackApi().trackGet(options)),
   );
 }
+
+export function useTrackIDWithLanguage(
+  lang: string,
+  options: TrackIdGetRequest = { id: -1 },
```

Comment:
it was a leftover because I played around with auto-completion and copilot chat (I guess Copilot is silver, thinking is gold ;-) )

I will commit a fix in a new PR

Reply:
Fixed in #323